src/content/self-hosted/gke.mdx

```diff
@@ -236,6 +166,9 @@ You'll need to take the `EXTERNAL-IP` address, and add it to your DNS for the do
 
 ## Sign in to your Okteto instance
 
-> **Important**: The default installation uses self-signed certificates. We strongly recommend that you  configure Okteto to [bring your own certificates](self-hosted/administration/certificates.mdx) before giving your team access to your instance.
+After a successful installation, you can access your Okteto instance at `https://okteto.SUBDOMAIN`. Your account will be automatically created as part of the login process. The first user to successfully login into the instance will be automatically assigned the `administrator` role.
+
+> **Important**: The default installation is not recommended for production use. We highly advise following the steps outlined in the [Admin Dashboard](administration/dashboard.mdx) before giving your team access to your Okteto instance.
 
-You can access your Okteto instance at `https://okteto.SUBDOMAIN`. Your account will be automatically created as part of the login process. The first user to successfully log into the instance will be automatically assigned the `administrator` role.
+The default installation is all you need to try our [getting started](getting-started.mdx) guide.
```

Comment:
@omnomagonz what do you think about the final CTA? and its tone?

Reply:
```suggestion
The default installation is all that's required to follow our [getting started](getting-started.mdx) guide.
```

Minor, non-blocking suggestion, but otherwise it sounds good to me 👍